benchutils/metrics.py

```diff
@@ -43,6 +43,8 @@ def correlation(observed, expected):
         The Pearson correlation between the series
 
     """
+    observed = np.array(observed)
```

Comment:
If parameters are already `np.array` (according to docstring), this conversion may be unnecessary.

Reply:
Yeah this is leftover from debugging. Will remove.